main.tf

```diff
@@ -30,32 +28,40 @@ resource "google_compute_instance" "gitlab" {
     }
 
     network_interface {
-      subnetwork = "${var.infra_name}-offensive-pipeline-subnet"
+      subnetwork = module.gcp-network.subnets_names[0]
```

Comment:
Can't you use a self link in this case?

Reply:
Yes:
https://github.com/SygniaLabs/ScallOps/commit/68b25391f48853d3f35a286fffd747d6fdf0e15d#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbL31